gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -54,9 +86,26 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
+  const std::string json_output_dir = absl::GetFlag(FLAGS_json_output_dir);
   const std::string output_dir = absl::GetFlag(FLAGS_output_dir);
-  if (output_dir.empty()) {
-    std::cerr << "Error: --output_dir is required\n";
+  const std::vector<std::string> output_types =
+      absl::GetFlag(FLAGS_output_types);
+  bool json_output_enabled = false;
+  bool asm_output_enabled = false;
+  for (const std::string& output_type : output_types) {
+    if (output_type == "json")
+      json_output_enabled = true;
+    else if (output_type == "asm")
+      asm_output_enabled = true;
+  }
```

Comment:
Simpler alternative if we do want to allow both types of output at the same time: just have two `bool` flags "FLAGS_json_output" and "FLAGS_asm_output".

Or even simpler: since we have different flags for output dirs, all we really need is:
- `FLAGS_json_output_dir` and `FLAGS_asm_output_dir`, and enable the output when the flag is non-empty. The output types can then be completely removed, and we wouldn't even need this check.

Reply:
Good point. Switched to just using the output directories.